src/Symfony/Bundle/FrameworkBundle/Tests/Command/CachePoolDeleteCommandTest.php

```diff
@@ -90,7 +90,12 @@ public function testCommandDeleteFailed()
     public function testComplete(array $input, array $expectedSuggestions)
     {
         $application = new Application($this->getKernel());
-        $application->add(new CachePoolDeleteCommand(new Psr6CacheClearer(['foo' => $this->cachePool]), ['foo']));
+        $command = new CachePoolDeleteCommand(new Psr6CacheClearer(['foo' => $this->cachePool]), ['foo']);
+        if (method_exists($application, 'addCommand')) {
```

Comment:
As this uses the Application class of FrameworkBundle inside the test of FrameworkBundle, the method is guaranteed to exist (as you are adding it in the same PR)

Reply:
same for all tests in FrameworkBundle (all least all the tests using the FrameworkBundle Application)